transports/transport-rest/transport-rest-client/src/main/java/org/apache/servicecomb/transport/rest/client/RestTransportClient.java

```diff
@@ -73,31 +73,23 @@ public void init(Vertx vertx) throws Exception {
     VertxUtils.blockDeploy(vertx, ClientVerticle.class, deployOptionshttp2);
   }
 
-  private static HttpClientOptions createHttpClientOptions() {
+  private static HttpClientOptions createHttpClientOptions(boolean http2) {
     HttpClientOptions httpClientOptions = new HttpClientOptions();
-    httpClientOptions.setMaxPoolSize(TransportClientConfig.getConnectionMaxPoolSize())
-        .setIdleTimeout(TransportClientConfig.getConnectionIdleTimeoutInSeconds())
+    httpClientOptions.setIdleTimeout(TransportClientConfig.getConnectionIdleTimeoutInSeconds())
```

Comment:
setIdleTimeout should use different configuration for http and http2

Reply:
done